apps/src/StudioApp.js

```diff
@@ -344,8 +344,8 @@ StudioApp.prototype.init = function (config) {
   ReactDOM.render(
     <Provider store={this.reduxStore}>
       <InstructionsDialogWrapper
-        showInstructionsDialog={(autoClose, showHints) => {
-            this.showInstructionsDialog_(config.level, autoClose, showHints);
+        showInstructionsDialog={(autoClose) => {
+            this.showInstructionsDialog_(config.level, autoClose);
```

Comment:
It looks like the `showHints` prop can also be removed from [InstructionsDialogWrapper.jsx](https://github.com/code-dot-org/code-dot-org/blob/0832e62f347eff6635556b6ac02270fd6c88e7e7/apps/src/templates/instructions/InstructionsDialogWrapper.jsx#L17)?